reinforcement_learning/webapps/vizualisation/webapp.json

```diff
@@ -0,0 +1,49 @@
+// This file is the descriptor for webapp vizualisation
+{
+    "meta": {
+        "label": "vizualisation",
+        "description": "Vizualize your agent playing",
```

Comment:
Bad description, remember that users who will see this webapp in the list will have no idea about anything, they are just users on a DSS instance. Take a step back and write a concise description that is short yet relatively understandable out of context by someone who does not know your topic.